documentation/modules/auxiliary/scanner/snmp/snmp_enum.md

```diff
@@ -0,0 +1,78 @@
+## Description
+This module performs a detailed enumeration of a host or a range through SNMP protocol. It supports hardware, software, and network information.
+
```

Comment:
I would call out the items which are pulled by this module in a list. IE:

1. `1.3.6.1.2.1.1.5.0` [sysName](http://www.oid-info.com/get/1.3.6.1.2.1.1.5)

Reply:
Sorry for such a late reply, will do the necessary changes asap.

Reply:
no worries!  I haven't had time yet to look at the other submissions anyways.